### ISSUE TITLE
[BC 7.2.5] Localization strings missing

### ISSUE THREAD

**Issue by maqjav** [BuildCraft/BuildCraft#3289]
There are some strings which aren't present in the lang files so they cannot be translated properly.

For example:

Configuration:
Experimental tab:

kinesisCanExplode
kinesisLossMode
kinesisPowerLossOnTravel
General tab:

builderMaxIterationsPerItemFactor
fuel.lava.combustion (didn't this one disappear?)
fuel.lava.combustion.energyOutput. (didn't this one disappear?)
Etc.

I was updating the spanish localization file and I also realized that not all the strings are extracted to the properties files, so for example the toolTips of every configuration option are in english (and they aren't in the en_US.lang file, so probably they are in the code?).

Thanks!

**Comment by AlexIIL** [comment]
Yeah, currently all config option comments are hardcoded in code. This can be fixed for BC 8 though.

If you are referring to the config keys (the bit before "=") then they won't be localized- it makes configs difficult to share and causes worries- for example, when you first change your language away from the default all of your existing configs will be ignored, and new ones will be generated.

**Comment by maqjav** [comment]
Oh no, I don't mean that, it will make coding very tricky!
I mean that if you access to the config panel (Configuration -> Experimental, for example) you will see that there are some options without text (it shows the property key instead of the value)

Here you can see what I mean:
![1](https://cloud.githubusercontent.com/assets/4415709/14327252/cdf203b8-fc31-11e5-8856-5df309aa3f18.PNG)

![2](https://cloud.githubusercontent.com/assets/4415709/14327263/d822b45e-fc31-11e5-8c80-102a917c575c.PNG)

**Comment by AEnterprise** [comment]
it shows the property key, use that to localize the strings, they are not localized in english but that doesn't mean you can't localize them in another language

**Comment by maqjav** [comment]
Aha... I think I will wait anyway until they are in english, the meaning seems obvious but just in case I missunderstand something I prefere to wait for the correct localization.

**Comment by AEnterprise** [comment]
well the meaning is pretty obvious so just translate them, if it's not 100% correct they can be updated later

**Comment by AlexIIL** [comment]
These should be in the english localization file anyway. I'll translate them in english to make sure it is the master file.

**Comment by AEnterprise** [comment]
not all of them are in the master file, should i add them?

**Comment by AlexIIL** [comment]
That would be very helpful @AEnterprise :)

**Comment by AEnterprise** [comment]
alright, expect a PR soon then
